### PR TITLE
Add pg_repack support to PG 12 and 13

### PR DESCRIPTION
**Pull request by joshraker**
Needed to upgrade a Database already using `pg_repack`

### CODE REVIEW

**Comment by joshraker** [comment]
Travis checks have been weird lately. The other day it tested against the wrong commit and this one wouldn't work until I force pushed a new commit.

File: 12-contrib/install-extras.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ apt-key add /tmp/GPGkeys/pglogical.key
 # Install packaged extensions first
 apt-install "^postgresql-plperl-${PG_VERSION}$" "^pgagent$" "^postgresql-${PG_VERSION}-pglogical$" \
   "^postgresql-plpython3-${PG_VERSION}$" "^postgresql-${PG_VERSION}-wal2json$" \
-  "^postgresql-${PG_VERSION}-pgaudit$"
+  "^postgresql-${PG_VERSION}-pgaudit$" "^postgresql-${PG_VERSION}-repack"
```
Comment (UserNotFound) [inline 12-contrib/install-extras.sh:14]:
```suggestion
  "^postgresql-${PG_VERSION}-pgaudit$" "^postgresql-${PG_VERSION}-repack$"
```

File: 13-contrib/install-extras.sh [hunk 2]
```diff
@@ -9,4 +9,5 @@ echo "deb [arch=amd64] http://packages.2ndquadrant.com/pglogical/apt/ stretch-2n
 apt-key add /tmp/GPGkeys/pglogical.key
 
 # Install packaged extensions first
-apt-install "^postgresql-${PG_VERSION}-pglogical$" "^pgagent$" "^postgresql-${PG_VERSION}-pgaudit$"
+apt-install "^postgresql-${PG_VERSION}-pglogical$" "^pgagent$" "^postgresql-${PG_VERSION}-pgaudit$" \
+            "^postgresql-${PG_VERSION}-repack"
```
Comment (UserNotFound) [inline 13-contrib/install-extras.sh:13]:
```suggestion
            "^postgresql-${PG_VERSION}-repack$"
```

**Review by UserNotFound — APPROVED** [review]
To be safe, we should continue to anchor the regex with `$`

File: 12-contrib/install-extras.sh (hunk 1 above)
Comment (joshraker) [reply 12-contrib/install-extras.sh:14]: Yup broken copy-paste, thanks

**Pull request merged**